packages/@aws-cdk/aws-stepfunctions-tasks/test/sagemaker/create-transform-job.test.ts

```diff
@@ -134,6 +134,11 @@ test('create complex transform job', () => {
     },
     maxConcurrentTransforms: 3,
     maxPayload: cdk.Size.mebibytes(100),
+    dataProcessing: {
+      inputFilter: '$[1:]',
+      joinSource: 'Input',
+      outputFilter: '$[0,-1]',
+    },
```

Comment:
do not modifying existing tests as it can mask regression when we update them.

to test your scenario: 
* create a new test that is minimal (required properties) and the capability you're adding (`dataProcessing`)